src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<Fruit, Integer> fruitsStorage = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<Fruit, Integer> fruitStorage = new HashMap<>();
```
fruit here is more of an adjective and IMO this is prone to misspelling, however it's up to you